swarm/src/main/resources/swarm/swarm.bom

```diff
@@ -53,7 +53,7 @@ brooklyn.catalog:
           description: |
             Size of the etcd cluster when created initially
           type: integer
-          default: 1
+          default: 2
```

Comment:
I don't think this is the case but it is recommended to use odd numbers - I'll up it to 3.